iothub_client/samples/pnp/pnp_simple_thermostat/CMakeLists.txt

```diff
@@ -0,0 +1,33 @@
+#Copyright (c) Microsoft. All rights reserved.
+#Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#this is CMakeLists.txt for pnp_simple_thermostat
+
+compileAsC99()
+
+set(pnp_simple_thermostat_c_files
+    pnp_simple_thermostat.c
+	../common/pnp_device_client_helpers.c
```

Comment:
Simple thermostat should not have any dependency on the facade/client_helpers.

Reply:
Fixed.